qupath-core/src/main/java/qupath/lib/roi/RoiTools.java

```diff
@@ -1074,6 +1074,9 @@ public static ROI createRandomRectangle(ROI mask, double width, double height, i
 		if (random == null)
 			random = new Random();
 
+		if (mask instanceof RectangleROI rectangleROI) {
```

Comment:
This isn't identical... values need to be integers with `ImageRegion`, but `double` with `RectangleROI`.